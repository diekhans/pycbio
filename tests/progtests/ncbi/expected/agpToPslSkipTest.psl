92954	0	0	0	0	0	0	0	-	000996F_1_92859_quiver_pilon	92954	0	92954	CM009248.2	129809613	0	92954	1	92954,	0,	0,
1318120	0	0	0	0	0	0	0	-	000346F_1_1318741_quiver_pilon	1318120	0	1318120	CM009248.2	129809613	102663	1420783	1	1318120,	0,	102663,
6000670	0	0	0	0	0	0	0	-	000145F_1_5996913_quiver_pilon	6000670	0	6000670	CM009248.2	129809613	1494817	7495487	1	6000670,	0,	1494817,
7054879	0	0	0	0	0	0	0	-	000131F_1_7046826_quiver_pilon	7054879	0	7054879	CM009248.2	129809613	7495587	14550466	1	7054879,	0,	7495587,
696527	0	0	0	0	0	0	0	+	000096F_1_9726372_s_1_696527	696527	0	696527	CM009248.2	129809613	14550566	15247093	1	696527,	0,	14550566,
4018217	0	0	0	0	0	0	0	-	000199F_1_4013115_quiver_pilon	4018217	0	4018217	CM009248.2	129809613	15371823	19390040	1	4018217,	0,	15371823,
1207857	0	0	0	0	0	0	0	-	000358F_1_1206579_quiver_pilon	1207857	0	1207857	CM009248.2	129809613	19390140	20597997	1	1207857,	0,	19390140,
17957201	0	0	0	0	0	0	0	-	000044F_1_17938287_quiver_pilon	17957201	0	17957201	CM009248.2	129809613	20598097	38555298	1	17957201,	0,	20598097,
939444	0	0	0	0	0	0	0	-	000394F_1_937897_quiver_pilon	939444	0	939444	CM009248.2	129809613	38555398	39494842	1	939444,	0,	38555398,
2218774	0	0	0	0	0	0	0	+	000288F_1_2215942_quiver_pilon	2218774	0	2218774	CM009248.2	129809613	39494942	41713716	1	2218774,	0,	39494942,
400731	0	0	0	0	0	0	0	-	000507F_1_400064_quiver_pilon	400731	0	400731	CM009248.2	129809613	41713816	42114547	1	400731,	0,	41713816,
83868	0	0	0	0	0	0	0	-	001085F_1_83613_quiver_pilon	83868	0	83868	CM009248.2	129809613	42114647	42198515	1	83868,	0,	42114647,
111253	0	0	0	0	0	0	0	+	000937F_1_110928_quiver_pilon	111253	0	111253	CM009248.2	129809613	42198615	42309868	1	111253,	0,	42198615,
242383	0	0	0	0	0	0	0	+	000604F_1_242084_quiver_pilon	242383	0	242383	CM009248.2	129809613	42309955	42552338	1	242383,	0,	42309955,
117266	0	0	0	0	0	0	0	+	000897F_1_116974_quiver_pilon	117266	0	117266	CM009248.2	129809613	42552438	42669704	1	117266,	0,	42552438,
105621	0	0	0	0	0	0	0	-	000964F_1_105257_quiver_pilon	105621	0	105621	CM009248.2	129809613	42669804	42775425	1	105621,	0,	42669804,
50968	0	0	0	0	0	0	0	-	001802F_1_51328_quiver_pilon	50968	0	50968	CM009248.2	129809613	42775525	42826493	1	50968,	0,	42775525,
218802	0	0	0	0	0	0	0	-	000618F_1_218458_quiver_pilon	218802	0	218802	CM009248.2	129809613	42847895	43066697	1	218802,	0,	42847895,
72633	0	0	0	0	0	0	0	+	001348F_1_72386_quiver_pilon	72633	0	72633	CM009248.2	129809613	43093709	43166342	1	72633,	0,	43093709,
132493	0	0	0	0	0	0	0	-	000818F_1_132298_quiver_pilon	132493	0	132493	CM009248.2	129809613	43166442	43298935	1	132493,	0,	43166442,
2339445	0	0	0	0	0	0	0	+	000280F_1_2337520_quiver_pilon	2339445	0	2339445	CM009248.2	129809613	43479164	45818609	1	2339445,	0,	43479164,
552447	0	0	0	0	0	0	0	-	000420F_1_772407_s_220982_773428	552447	0	552447	CM009248.2	129809613	46060225	46612672	1	552447,	0,	46060225,
28549075	0	0	0	0	0	0	0	-	000013F_1_28518784_quiver_pilon	28549075	0	28549075	CM009248.2	129809613	46702439	75251514	1	28549075,	0,	46702439,
177194	0	0	0	0	0	0	0	-	000701F_1_177051_quiver_pilon	177194	0	177194	CM009248.2	129809613	75251614	75428808	1	177194,	0,	75251614,
185540	0	0	0	0	0	0	0	+	000420F_1_772407_s_1_185540	185540	0	185540	CM009248.2	129809613	75428908	75614448	1	185540,	0,	75428908,
2120133	0	0	0	0	0	0	0	-	000300F_1_2117096_quiver_pilon	2120133	0	2120133	CM009248.2	129809613	75711145	77831278	1	2120133,	0,	75711145,
36877801	0	0	0	0	0	0	0	-	000006F_1_36840141_quiver_pilon	36877801	0	36877801	CM009248.2	129809613	83435047	120312848	1	36877801,	0,	83435047,
2967433	0	0	0	0	0	0	0	-	000251F_1_2964467_quiver_pilon	2967433	0	2967433	CM009248.2	129809613	120312914	123280347	1	2967433,	0,	120312914,
4492301	0	0	0	0	0	0	0	-	000181F_1_4490123_quiver_pilon	4492301	0	4492301	CM009248.2	129809613	123324154	127816455	1	4492301,	0,	123324154,
508979	0	0	0	0	0	0	0	+	000468F_1_508232_quiver_pilon	508979	0	508979	CM009248.2	129809613	127816555	128325534	1	508979,	0,	127816555,
1176964	0	0	0	0	0	0	0	-	000362F_1_1179733_quiver_pilon	1176964	0	1176964	CM009248.2	129809613	128632649	129809613	1	1176964,	0,	128632649,
