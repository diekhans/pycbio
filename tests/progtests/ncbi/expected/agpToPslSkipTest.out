Warning name 000171F_1_5102983_quiver_pilon not found in component size map
Warning name NBAG03000714.1 not found in component local id map
